src/content/docs/patch-signing.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Patch Signing (preview)
+description: How to sign patches
+---
+
+{/* cspell:words outform pubout */}
+
+:::note
+We've shipped this preview of patch signing to unblock certain customer
+evaluations. While we're confident that this preview adds a useful layer of
+security, it does not yet have all the expected integrations with cloud signing,
+keychains, etc. that a final product would expect to have. We also do not yet
+manage signing keys for you or support many key formats or signature types.
+Signed patches are safe and ready for use, but may have rough edges or missing
+conveniences. Any customer should feel welcome to use this preview in
+production, and is encouraged to file feedback for any rough edges you may find.
```

Comment:
```suggestion
production, and is encouraged to file feedback for any bumps you may find.
```
Or something else, just so we don't repeat `rough edges`

Reply:
Done